src/deployment/registration.py

```diff
@@ -504,6 +504,21 @@ def assign_scaleset_role(onefuzz_instance_name: str, scaleset_name: str) -> None
         assign_scaleset_role_manually(onefuzz_instance_name, scaleset_name)
 
 
+def assign_multi_tenant_auth(objectId: str) -> None:
+
+    try:
+        query_microsoft_graph(
+            method="PATCH",
+            resource="applications/%s" % objectId,
+            body={"signInAudience": "AzureADMultipleOrgs"},
+        )
+    except adal.AdalError:
+        raise Exception(
+            "error setting signInAudience in ad application %s: %s"
+            % (objectId, adal.AdalError)
+        )
```

Comment:
In this case can you print out the command to make the assignment see `assign_scaleset_role_manually`

Reply:
Yeah, I noticed you did that. Why did you do that? If the "try" block fails then asking the user to run the command again right after this operation failed, isn't going to change the output? 

Regardless, I'm happy to add a similar line: "execute the following query in the azure portal bash shell: "
Are you suggesting I put the print line and replace the 'raise Exception' call?

Reply:
We had an issue recently on linux machines that could not execute the registration code. We added this as a mitigation strategy to allow people to complete the registration in that case. 
Yes the suggestion is to replace the raise excepetion unless this registration is required for the rest of the script to continue.

Reply:
Thanks for pointing this out. I was also using the wrong exception. So, my updated changes will require the user to manually make the change if the azure graph call fails, otherwise deployment will not continue until the setting is enabled. This will provide better support for Linux users, where the azure graph call fails.